CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,83 @@
+## Goal
+
+Our goal is to provide a space where it is safe for everyone to contribute to
+open-source software in a respectful and peaceful manner.
+
+We value all contributions and want to make this organization and its surrounding community
+a place for everyone.
+
+As members, contributors and everyone else who may participate in the development, we strive
+to keep the entire experience civil.
+
+## Standards
+
+Our community is made for everyone to contribute to the codebase together.
+
+Our standards are:
+ - Do not harass, attack, or in any other way discriminate against anyone, including for their protected traits, including, but not limited to, sex, religion, race, appearance, gender, identity, nationality, sexuality, etc.
+ - Do not go off-topic, do not post spam.
+ - You are not required to like everybody, but you are required to treat everyone with respect.
+
+Examples of breaking each rule respectively include:
+ - Harassment, bullying or joking about someone's protected trait.
+ - Posting distasteful imagery, trolling, or political discussions.
+ - Treating someone as worse because of their lack of understanding of an issue.
+
+## Enforcement
+
+Enforcement of this CoC is done by the members of the hyprwm organization.
+
+We, as the organization, will strive our best to keep this community civil and
+following the standards outlined above.
+
+### Reporting incidents
+
+If you believe an incident of breaking our standards has occurred, but nobody has
+taken appropriate action, you can privately contact the people responsible for dealing
+with such incidents in multiple ways:
+
+***E-Mail***
+ - `vaxry[at]vaxry.net`
+ - `mihai[at]fufexan.net`
+
+***Discord***
+ - `@vaxry`
+ - `@fufexan`
+
+***Matrix***
+ - `@vaxry:matrix.vaxry.net`
+ - `@fufexan:matrix.org`
```

Comment:
You may also want to add:

* A means of anonymous appeal
* An outline of the protections afforded to those involved in conduct-related incidents. I recommend guaranteeing the privacy of both the accuser and the accused.

Reply:
re point 1: we don't have such a means, you can use a temp email I guess.